src/common/datadog.config.js

```diff
@@ -0,0 +1,18 @@
+import { datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+    applicationId: '3937053e-386b-4b5b-ab4a-c83217d2f953',
+    clientToken: process.env.DATADOG_CLIENT_TOKEN,
+    site: 'datadoghq.com',
+    service:'give-web',
+    env: process.env.NODE_ENV,
+    version: process.env.GITHUB_SHA,
+    sessionSampleRate:100,
+    sessionReplaySampleRate: 20,
```

Comment:
@mattdrees do you have an opinion on the values of these rates to start off with? This would be 100% of sessions are recorded and 20% of those are set up for the session replay feature.

Reply:
These were the defaults that DataDog suggested. And it is what MPDx React is using.

Reply:
My opinion would be to start with both of those at 100 for nonproduction. And for production, to start with 10% for sessionSampleRate. For `sessionReplaySampleRate` in prod, I think I'd go with 10% (which means 1% of all users). 

I think the goal here would be to keep browser rum costs for givesite below about $100 / month. So start with those settings (or something similar), and then adjust a month later once we see how costs turn out.